src/lang/english.txt

```diff
@@ -3403,7 +3415,9 @@ STR_VEHICLE_LIST_ROAD_VEHICLE_TOOLTIP                           :{BLACK}Road veh
 STR_VEHICLE_LIST_SHIP_TOOLTIP                                   :{BLACK}Ships - click on ship for information
 STR_VEHICLE_LIST_AIRCRAFT_TOOLTIP                               :{BLACK}Aircraft - click on aircraft for information
 
+STR_VEHICLE_LIST_PROFIT_EFFICIENCY_LINE                         :{STRING2}   {STRING2}
```

Comment:
I'm not sure this is really the best way to present the data in the vehicle window. Should I just put it on two lines instead, making the details window taller?

Reply:
Trying on a separate line instead:
![image](https://user-images.githubusercontent.com/1062071/55901554-d57ffe00-5bc9-11e9-9cf1-00dfa21a61e1.png)

Reply:
Going with 5 lines after a bit of discussion.